### ISSUE TITLE
Search crushes app

### ISSUE THREAD

**Issue by 4eb0da** [gemini-testing/html-reporter#190]
```
TypeError: "k.shouldSuiteBeShown is not a function"

    f http://localhost:8000/gui.min.js:60
    value http://localhost:8000/gui.min.js:60
    Pn http://localhost:8000/gui.min.js:31
    Nn http://localhost:8000/gui.min.js:31
    Mn http://localhost:8000/gui.min.js:31
    Xn http://localhost:8000/gui.min.js:31
    Qn http://localhost:8000/gui.min.js:31
    vr http://localhost:8000/gui.min.js:31
    dr http://localhost:8000/gui.min.js:31
    Or http://localhost:8000/gui.min.js:31
    ze http://localhost:8000/gui.min.js:31 
```

Steps to reproduce:
1) Create project by "quick start" guide https://github.com/gemini-testing/hermione#quick-start
2) Add html-reporter plugin
```js
    plugins: {
        'html-reporter/hermione': {}
    }
```
3) Run hermione: `node_modules/.bin/hermione gui`
4) Input "o" in search field
5) Click "github" test case

**Comment by j0tunn** [comment]
@CatWithApple tale a look please

**Comment by CatWithApple** [comment]
looking right now

**Comment by CatWithApple** [comment]
Will be fixed in this PR:
https://github.com/gemini-testing/html-reporter/pull/189

Changes:
https://github.com/gemini-testing/html-reporter/pull/189/files#diff-7b568f8b160b4e3ddf7c3b7ab004d1cbR9

@Flackus look

**Comment by Flackus** [comment]
Fixed in `html-reporter@3.4.2`.

@4eb0da thanks for the report!

**Comment by 4eb0da** [comment]
Cannot reproduce with `3.4.2`
Thx for instant response & fix!
